rmf_traffic_ros2/src/rmf_traffic_ros2/schedule/MonitorNode.hpp

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#ifndef SRC__RMF_TRAFFIC_SCHEDULE__MONITORNODE_HPP
```

Comment:
Nitpick about the header name:

Since this class has both a public and private header related to it:
* `include/rmf_traffic_ros2/schedule/MonitorNode.hpp`
* `src/rmf_traffic_ros2/schedule/MonitorNode.hpp`

I prefer to add `internal_` to the file name of the header that isn't part of the public API (e.g. `src/rmf_traffic_ros2/schedule/internal_MonitorNode.hpp`). I think that helps communicate at a glance that the file you're looking at is specifically the internal header, and not the public API header. It can also help make `#include` directives and file searches less confusing.

Reply:
Renamed the internal header in b65c743.